guides/chalet_time_policy.md

```diff
@@ -102,11 +102,11 @@ Scheduling team will giving the Talent team coordinators access to the Kimble re
 
 #### 4. Research and development
 
-The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the #chalettimeteam Slack channel at the beginning of each week. 
+The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the [`#chalet-time-team`](https://madetechteam.slack.com/archives/C03F23K2RL0) Slack channel at the beginning of each week. 
```

Comment:
Potentially useful information, markdown has a concept of [Reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) which would reduce the number of places this URL was repeated.